portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -2005,6 +2289,57 @@ definition {
 		}
 	}
 
+	@description = "LPS-124960 When there's a WebContent1 of type Subtype1 and WebContent2 of type Subtype2, and the user filters by Subtype1 and Subtype2, both web content are shown in the list"
+	@priority = "3"
+	test FilterByMultipleTypes {
```

Comment:
Would you mind changing to FilterByTypes to align with FilterByCategories, FilterByCategoryAndAuthors, and so on?